app/src/main/java/com/example/themealsapp/presentation/view/MealSearchFragment.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.themealsapp.presentation.view
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.themealsapp.R
+import com.example.themealsapp.databinding.FragmentMealSearchBinding
+import com.example.themealsapp.domain.model.Meal
+import com.example.themealsapp.presentation.view.adapter.MealsListAdapter
+import com.example.themealsapp.utils.BaseFragment
+import com.example.themealsapp.utils.UIState
+
+private const val TAG = "MealSearchFragment"
+class MealSearchFragment : BaseFragment() {
+
+    private val binding by lazy {
+        FragmentMealSearchBinding.inflate(layoutInflater)
+    }
+
+    private val mealsListAdapter by lazy {
+        MealsListAdapter {
+
+        }
+    }
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        binding.rvMealSearch.apply {
+            layoutManager = LinearLayoutManager(
+                requireContext(),
```

Comment:
why are you using require context instead of context or application context?

Reply:
because I'm currently working on top of the activity. Pulling the application context would be too unnecessary.